### PR TITLE
Modify editor for being able to set revert and save logic. Add sending test copy in editor. Modify steps with templateEditor in form and draft wizards

### PR DESCRIPTION
**Pull request by radiansz**

### CODE REVIEW

**Comment by trikashny** [comment]
"Backspace bug" in the input field. 

Error in console: 

> sendTestCopy.js?version=1471259105528:31 Uncaught TypeError: Cannot read property 'sendTestCopy' of undefined`

This merge shall not pass!

**Comment by radiansz** [comment]
@trikashny I need more details. I failed to achieve same result

File: src/blocks/shared/WizardEditorFooter/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+        _handleSavePress: function(event) {
+            var buttonEl = event.currentTarget;
+            buttonEl.classList.add('loading');
+
+            this._saveAndClose().then(function() {
+                buttonEl.classList.remove('loading');
+            });
+        },
+
```
Comment (rosendi) [inline src/blocks/shared/WizardEditorFooter/index.js]: \n

File: src/blocks/shared/WizardEditorFooter/index.less [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+.WizardEditorFooter__leftSide > * {
+    margin-right: 10px;
+}
+
+.WizardEditorFooter__rightSide > * {
+    margin-left: 10px;
+}
+
+
+.WizardEditorFooter_hidden {
```
Comment (rosendi) [inline src/blocks/shared/WizardEditorFooter/index.less]: remove

File: src/blocks/templateEditor/templateEditor.js [hunk 3]
```diff
@@ -134,13 +134,7 @@ define(function(require) {
...
                 activeBlock && activeBlock.deactivate();
             },
-            'close .templateEditorBottomPanel': function() {
-                this.el.querySelector('.draftPreview').block.setIFrameContents(this.models.template.compile());
-                this.setModifiers({
-                    fullscreen: false,
-                    preview: true
-                });
-            },
+            'close .templateEditorBottomPanel': 'minimizeEditor',
```
Comment (rosendi) [inline src/blocks/templateEditor/templateEditor.js]: remove

File: src/blocks/templateEditor/templateEditor.less [hunk 4]
```diff
@@ -171,3 +171,12 @@
     color: black;
     padding: 10px 20px;
 }
+
+.templateEditor__footer {
+
```
Comment (rosendi) [inline src/blocks/templateEditor/templateEditor.less]: remove the new line

File: src/resources/presentation/draft/block/model.js [hunk 5]
```diff
@@ -72,6 +78,15 @@ define(function(require) {
             this.set('blocks', unions);
 
             this.initJSONSettings(json);
+
+            this.updateInitialState();
+        },
+
+        revert: function() {
+            var oldState = JSON.parse(JSON.stringify(this.get('initialState')));
```
Comment (rosendi) [inline src/resources/presentation/draft/block/model.js]: rename to `prevState`

**Comment by rosendi** [comment]
Fix `main.css`:

```
GET http://0.0.0.0:8080/blocks/htmlEditor/htmlEditorBottomPanel/htmlEditorBottomPanel.css 
GET http://0.0.0.0:8080/blocks/templateEditor/templateEditorBottomPanel/templateEditorBottomPanel.css 
```

**Comment by rosendi** [comment]
LGTM

**Pull request merged**